src/pudl/transform/ferc1.py

```diff
@@ -32,6 +33,92 @@
 
 logger = get_logger(__name__)
 
+TABLES_LITERAL = Literal[
+    "plants_steam_ferc1",
+    "fuel_ferc1",
+]
```

Comment:
this is a very good idea! I've kind of been dumping these metadata dictionaries up here as a staging ground but don't really want them to stay here.

Reply:
.... But i think one of the name issues with pulling all of these renames into the metadata is that there are  some columns in the original tables that have very different units. It would feel semi-dangerous to rename them before converting the units or not tightly coupled with converting the units.

Reply:
I agree generic transform metadata should live with the rest of the Pydantic metadata to avoid duplication. I'm not sure how to design the transform functions to access the metadata in a clean way. I like Zane's first option because the actual parameterization of the rename function is hidden within a method in a `Resource` which I think would make it dagster friendly. Does it make sense to keep adding new methods to Resource for each new generic transform function? What about bespoke cleaning functions for a particular table?